mock/mockimpl/kvproc/operations.go

```diff
@@ -816,8 +816,9 @@ type MultiLookupOptions struct {
 
 // MultiLookupResult contains the results of a SD_MULTILOOKUP operation.
 type MultiLookupResult struct {
-	Cas uint64
-	Ops []*SubDocResult
+	Cas          uint64
+	Ops          []*SubDocResult
+	DocIsDeleted bool
```

Comment:
`IsDeleted`?

Reply:
I felt like the result itself couldn't be deleted so that didn't make sense but at the same time the result itself can't be deleted so it has to refer to the document and does make sense.